Task_Position_the_Cubes/index.html

```diff
@@ -1,15 +1,22 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>Cubes Problem</title>
-    <link rel="stylesheet" href="index.css" />
-  </head>
-
-  <body style="margin: 0; padding: 0">
-    <div top="0">
-      <a href="../index.html">Go Back</a>
-    </div>
-
```

Comment:
Consider adding content inside the <div> elements with classes 'child one', 'child two', etc. to display the cubes.